Subject: [PATCH] Fix some annoyances
Maintainer: Miroslav Koškár <http://mkoskar.com/>
Version: v1

* get rid of flashing
* make tilde to be fg=color243

---
 src/itemview_formaction.cpp | 2 +-
 src/view.cpp                | 7 +++++--
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/src/itemview_formaction.cpp b/src/itemview_formaction.cpp
index 7d4818c..0f3b171 100644
--- a/src/itemview_formaction.cpp
+++ b/src/itemview_formaction.cpp
@@ -569,7 +569,7 @@ void itemview_formaction::set_regexmanager(regexmanager * r) {
 		attrstr.append(utils::strprintf("@style_%u_normal:%s ", i, it->c_str()));
 	}
 	attrstr.append("@style_b_normal[color_bold]:attr=bold @style_u_normal[color_underline]:attr=underline ");
-	std::string textview = utils::strprintf("{textview[article] style_normal[article]: style_end[styleend]:fg=blue,attr=bold %s .expand:vh offset[articleoffset]:0 richtext:1}", attrstr.c_str());
+	std::string textview = utils::strprintf("{textview[article] style_normal[article]: style_end[styleend]:fg=color243 %s .expand:vh offset[articleoffset]:0 richtext:1}", attrstr.c_str());
 	f->modify("article", "replace", textview);
 }

diff --git a/src/view.cpp b/src/view.cpp
index 6d0bb20..a40c9ec 100644
--- a/src/view.cpp
+++ b/src/view.cpp
@@ -321,7 +321,7 @@ void view::open_in_pager(const std::string& filename) {
 		cmdline.append(" ");
 		cmdline.append(filename);
 	}
-	stfl::reset();
+	//stfl::reset();
 	LOG(LOG_DEBUG, "view::open_in_browser: running `%s'", cmdline.c_str());
 	::system(cmdline.c_str());
 	pop_current_formaction();
@@ -349,7 +349,7 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append(utils::replace_all(url,"'", "%27"));
 		cmdline.append("'");
 	}
-	stfl::reset();
+	//stfl::reset();
 	LOG(LOG_DEBUG, "view::open_in_browser: running `%s'", cmdline.c_str());
 	::system(cmdline.c_str());
 	pop_current_formaction();
@@ -936,6 +936,7 @@ void view::apply_colors(std::tr1::shared_ptr<formaction> fa) {
 		if (fgcit->first == "article") {
 			if (fa->id() == "article" || fa->id() == "help") {
 				std::string styleend_str;
+				/*
 				if (bgcit->second != "default") {
 					styleend_str.append("bg=");
 					styleend_str.append(bgcit->second);
@@ -943,7 +944,9 @@ void view::apply_colors(std::tr1::shared_ptr<formaction> fa) {
 				if (styleend_str.length() > 0)
 					styleend_str.append(",");
 				styleend_str.append("attr=bold");
+				*/

+				styleend_str.append("fg=color243");
 				fa->get_form()->set("styleend", styleend_str.c_str());
 			}
 		}
